src/components/events/Event/index.js

```diff
@@ -45,6 +46,24 @@ module.exports = React.createClass({
     };
   },
 
+  activate: function() {
+    this.setState({ activeClass: 'active' });
+  },
+
+  activateAndTimeout: function() {
+    this.activate();
+    this.timeout();
+  },
+
+  timeout: function() {
+    var _this = this;
```

Comment:
`this` is ok by itself. In the scope of the function. But when assigning `this` to a var, the options are to use `_this`, `that`, `self` or some other variable that describes what `this` is. To be honest our convention of describing what `this` is was born out of a problem when using jQuery. Sometimes multiple `_this` where defined within a nested scope. So it wan't immediately clear which `_this` is the `this` you want. We decided it was just safer to say what `this` is when we need to assign it to use in a different scope.